service/src/main/java/io/camunda/service/transformers/filter/UserTaskFilterTransformer.java

```diff
@@ -72,13 +72,12 @@ public SearchQuery toSearchQuery(final UserTaskFilter filter) {
 
   @Override
   public List<String> toIndices(final UserTaskFilter filter) {
-    final var completed = filter.completed();
-    final var canceled = filter.canceled();
+    final var created = filter.created();
 
-    if (completed || canceled) {
-      return Arrays.asList("tasklist-task-8.5.0_alias");
-    } else {
+    if (created) {
       return Arrays.asList("tasklist-task-8.5.0_");
+    } else {
+      return Arrays.asList("tasklist-task-8.5.0_alias");
```

Comment:
❓ Understanding question: why do we have version numbers on those indices? Do we have to update those at some point or anything, e.g. before/after releasing version 8.6? 

💭 If not, can we remove the versions at least or go even further and establish an enum with TASKLIST-TASK, OPERATE-XYZ, or even just TASK, VARIABLE, and so on that is evaluated accordingly by the Query API later on, depending on where it goes to (ES, OS, maybe something else entirely at some point)? This looks quite error-prone and is hard to understand. It doesn't have to be in the scope of this PR here, it can be a follow-up issue (if it doesn't exist already).

Reply:
We have this discussion with @ThorbenLindhauer, We cant rename them, because it would imply on data migration, as we cant just rename them over ES/OS (sadly). 

We can have another round of discussion up to the future, as other indices have similar standard .

Reply:
I am fine with the name in general, or I don't care about it much, to be honest. But I would like to NOT have them referenced like this in the service layer code. This is leaking information from lower levels into the service layer. This could be an enum, used in the service layer and later resolved in the query API implementation accordingly.

Reply:
I agree with your point. I will create a follow up issue, because there are other services that use this (Process Instance, Variable) that it is necessary to do this.

Reply:
I think we should have place where to the spec of the indices and have a mapping Index-> Index name or alias, like we already have in Tasklist
this can be done later as you said

Reply:
Thanks @nathansandi and @houssain-barouni, sound good to me for starters 👍 I agree that it can be done later in a follow-up 👍